packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -194,19 +194,15 @@
         availableLanguages,
         availableLibraryCategories,
         searchableLabels,
-        activeSearchTerms,
```

Comment:
Seems like this was all copy pasted from another component in this folder - @nucleogenesis should we move the common things into the useBaseSearch composable instead?

Reply:
Yep definitely